templates/federated_reporting/transport.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/bash
+#
+# Transport dump files from the feeder hubs to the superhub.
+#
+# Can be run as:
+#   transport.sh
+#         On a feeder hub, pushes dump files to the superhub.
+#   transport.sh push
+#         The same as with no arguments.
+#   transport.sh pull FEEDER_HUB [FEEDER_HUB2...FEEDER_HUBn]
+#         On the superhub, pull dumps from the given feeder hubs (in parallel).
```

Comment:
@vpodzime @nickanderson I would avoid using the word dump here.

Reply:
- s/dumps/exported data/
- s/dumps/excretions/
- s/dumps/discharge/
?

Reply:
Not deeply concerned, to be honest, this is not really supposed to be run manually by people who don't know what a dump is. Besides, `pg_dump` is the tool that creates those files.